src/app-layer-protos.h

```diff
@@ -69,19 +69,18 @@ enum AppProtoEnum {
     // HTTP for any version (ALPROTO_HTTP1 (version 1) or ALPROTO_HTTP2)
     ALPROTO_HTTP,
 
-    /* used by the probing parser when alproto detection fails
-     * permanently for that particular stream */
-    ALPROTO_FAILED,
```

Comment:
why is this not left static? commit message doesn't explain

Reply:
Changed back to static in next version